buildlib/azure-pipelines-release.yml

```diff
@@ -41,3 +41,13 @@ stages:
               assets: |
                 ./ucx-*.tar.gz
                 ./rpm-dist/ucx-*.src.rpm
+
+          - bash: |
```

Comment:
Let's move this step into the template. It can depend on `target` value.

Reply:
`and(succeeded(), eq(${{ parameters.target }}, "publish"))` is it correct?

Reply:
It should be good.